Training/python/EventMix.py

```diff
@@ -0,0 +1,103 @@
+import ROOT
+import glob
+import os
+from makeTree import MakeTupleClass
+
+
+ROOT.gROOT.SetBatch(True) # Don't show window
+ROOT.ROOT.EnableImplicitMT(4) # multi thread (use 4 threads)
+_rootpath = os.path.abspath(os.path.dirname(__file__)+"../../..") 
+ROOT.gROOT.ProcessLine(".include "+_rootpath)
+
+# Define tau tuple class from one of the inputs
+class_def = MakeTupleClass('taus', "/home/russell/skimmed_tuples/MuTau_prod2018/SingleMuon_skimmed.root", 'input_tuple',
+               'Tau', 'TauTuple')
+ROOT.gInterpreter.ProcessLine(class_def)
+
+ROOT.gInterpreter.Declare(' # include "TauMLTools/Training/interface/DataMixer.h" ')
+
+# Number of events of each type desired per batch:
+n_data = 50
+n_DY_taus = 36
+n_tt_taus = 2
+n_W_jets = 5
+n_QCD_jets = 5
+n_tt_jets = 1
+n_DY_muons = 1
+
+print("Mix Characteristics:")
+
+
+dataDesc = ROOT.DatasetDesc()
+dataDesc.n_per_batch = n_data
+dataDesc.selection = "data"
+dataDesc.fileNames.push_back("/home/russell/skimmed_tuples/MuTau_prod2018/SingleMuon_skimmed.root")
```

Comment:
move input root files to the cenral eos storage and fix paths accordingly (here and below)